README.md

```diff
@@ -74,6 +74,20 @@ We are dedicated to building a welcoming, diverse, and safe community. We expect
 
 This repository is a monorepo managed by Lerna. This means that all components are centrally managed here, even though we publish them to NPM as separate packages.
 
+## Commits
+
+We are using [Conventional Commits](https://www.conventionalcommits.org/) to automatically version the components and update their changelog's.
+Feel free to use a [tool of your choice](https://www.conventionalcommits.org/en/about/#tooling-for-conventional-commits) to generate these commits.
+
+Example for a commit message
+```
+feat(button): add new color blue
+
+Closes #ticket-1234
```

Comment:
Hmmm.. does this force multi-line commit messages??? I use git CLI in a shell and never do multi-line....

Reply:
Yes it does. But should not be a problem. This is what I found on Google.

>Another method of adding a multi-line Git commit message is using quotes with your message, though it depends on your shell's capacity. To do this, add single or double quotes before typing the message, keep pressing enter and writing the next line, and finally close the quote at end of the message.